wasted-app/RestaurantRegistrationControl.cs

```diff
@@ -62,27 +63,22 @@ private void RegisterButton_Click(object sender, EventArgs e)
                 ServicesController controller = ServicesController.Instance;
                 var mail = mailTextBox.Text;
                 var password = passwordTextBox.Text;
-                var restaurant = controller.RestaurantService.GetByMail(new Mail(mail));
-                if (restaurant == null)
+                var restaurantName = restaurantNameTextBox.Text;
+                var latitude = Convert.ToDecimal(latitudeTextBox.Text);
+                var longitude = Convert.ToDecimal(longitudeTextBox.Text);
```

Comment:
program probably crashes if input is empty or non-numeric, right?

Reply:
yea but we decided to leave it for now, right?

Reply:
I will check maybe there is a quick fix for this in textbox parameters

Reply:
nope, by default textbox can't do any validation. you could check my latest closed pr to see how I did it, to only allow numbers to be put in  (#67 ), except it still crashes when it's empty.

Reply:
shouldn't crash now